### PR TITLE
Removed links to the Mobile Apps.

### LINKED ISSUE #17: Remove Links to download the mobile app everywhere!
image
Link to download mobile app is not given.
I think here we can target this to the end of the website where the download App option is available or we can add both the components for ios and android above
Share your thoughts

### PR DESCRIPTION
**Pull request by jainaayush01**
closes #17 
Also updated the browser list.

### CODE REVIEW

File: src/components/Shared/Footer/_About.js [hunk 1]
```diff
@@ -12,12 +12,8 @@ const FooterAbout = () => {
...
       </p>
       <div className="Footer-About-Vendors">
-        <a href="https://itunes.apple.com/us/app/easya/id1478549100">
-          <img src={FooterAppleLogo} alt="Apple" />
-        </a>
-        <a href="https://play.google.com/store/apps/details?id=io.easya.easya">
-          <img src={FooterGoogleLogo} alt="Google" />
-        </a>
+        <img src={FooterAppleLogo} alt="Apple" />
+        <img src={FooterGoogleLogo} alt="Google" />
```
Comment (praveenscience) [inline src/components/Shared/Footer/_About.js:16]: Is it displaying correctly without the `<a>` tag?

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please remove the `package-lock.json` from the PR.

File: src/components/Shared/Footer/_About.js (hunk 1 above)
Comment (jainaayush01) [reply src/components/Shared/Footer/_About.js:16]: Yes, it is displaying the exact same without a tag.
Keeping the <a href="#"> was showing warnings so I removed it and the result was same.
image

Comment (praveenscience) [reply src/components/Shared/Footer/_About.js:16]: Okay thanks. 👍

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please just remove the `package-lock.json` from this! 😊

**Review by praveenscience — APPROVED** [review]
👍

**Pull request merged**